CapuaEnvironment/ExecutionUnit/ExecutionUnit.py

```diff
@@ -547,8 +532,8 @@ def DIV(self, hardware=False):
 
         resultA = sourceValue // destinationValue
         resultB = sourceValue % destinationValue
-        self.eu.setRegisterValue(REGISTER_A, resultA)
-        self.eu.setRegisterValue(REGISTER_B, resultB)
+        self.eu.setRegisterValue(REGISTERS["A"], resultA)
+        self.eu.setRegisterValue(REGISTERS["B"], resultB)
```

Comment:
I'll need to review this underlying part of the core. We might be able to hit 2 birds with 1 stone with the above simplifications.

Reply:
Need to re-evaluate performance on that whole segment before doing anything more here.